### PR TITLE
update STATUS_id_handle hateffect_id_handle STATUS_id_handle

### PR DESCRIPTION
**Pull request by sctnightcore**
ok ♥

### CODE REVIEW

File: tables/STATUS_id_handle.txt [hunk 1]
```diff
@@ -901,6 +901,53 @@
...
+981 EFST_BREEZE_BEAD
+983 EFST_AID_PERIOD_RECEIVEITEM_2ND
+984 EFST_AID_PERIOD_PLUSEXP_2ND
+985 EFST_AID_PERIOD_PLUSJOBEXP_2ND
+986 EFST_PRONTERA_JP
+988 EFST_GLOOM_CARD
+989 EFST_PHARAOH_CARD
+990 EFST_KIEL_CARD
+992 EFST_CHEERUP
+995	EFST_S_MANAPOTION
```
Comment (lututui) [inline tables/STATUS_id_handle.txt]: There's a tab here

File: tables/STATUS_id_handle.txt [hunk 2]
```diff
@@ -901,6 +901,53 @@
 935 EFST_DORAM_BUF_01
 936 EFST_DORAM_BUF_02
 937 EFST_SPRITEMABLE
+#https://github.com/zackdreaver/ROenglishRE/blob/master/data/luafiles514/lua%20files/stateicon/efstids.lub
```
Comment (lututui) [inline tables/STATUS_id_handle.txt]: Maybe put this at the top of the file

Comment (sctnightcore) [reply tables/STATUS_id_handle.txt]: ok sir

File: tables/STATUS_id_handle.txt (hunk 1 above)
Comment (sctnightcore) [reply tables/STATUS_id_handle.txt]: wait  XD

**Pull request merged**